index.html

```diff
@@ -0,0 +1,383 @@
+                    <span>History</span>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">play_arrow</i>
+                    <span>Your Videos</span>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">watch_later</i>
+                    <span>Watch Later</span>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">thumb_up</i>
+                    <span>Liked Videos</span>
+                </div>
+            </div>
+            <hr />
+
+            <div class="sidebar__categories">
+                <div class="sidebar__category">
+                    <h4>SUBSCRIPTIONS</h4>
+                </div>
+                <div class="sidebar__category">
+                    <img src="images/Images__001.jpg" alt="">
+                    <span>Belay Birhanu</span>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">add_circle</i>
+                    <span>Browse Channels</span>
+                </div>
+            </div>
+            <hr />
+
+            <div class="sidebar__categories">
+                <div class="sidebar__category">
+                    <h4>EXPLORE</h4>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">sports_esports</i>
+                    <span>Gaming</span>
+                </div>
+                <div class="sidebar__category">
+                    <i class="material-icons">live_tv</i>
+                    <span>Browse Channels</span>
+                </div>
+
+            </div>
+        </div>
+
+        <!-- End of Sidebar -->
+
+        <div class="feed__container">
+            <div class="categories">
+                <section class="category__section">
+                    <button class="category active">All</button>
+                    <button class="category">Category 1</button>
+                    <button class="category">Category 2</button>
+                    <button class="category">Category 3</button>
+                    <button class="category">Category 4</button>
+                    <button class="category">Category 5</button>
+                    <button class="category">Category 6</button>
+                    <button class="category">Category 7</button>
+                    <i class="material-icons">navigate_next</i>
+                </section>
+            </div>
+            <!-- ENDE OF CATEGORY -->
+            <div class="videos">
+                <section class="video__section">
+                    <article class="video__container">
+                        <a href="#" class="thumbnail" data-duration="12:24">
+                            <!-- <img src="http://unsplash.it/250/150?gravity=center" alt="" class="thumbnail-image">
```

Comment:
Please do not leave unused codes in your codebase as a comment.

Reply:
- [ ] Greate job, your comments are very polite 👏. However, I think you can improve this comment by explain why they should not leave commented codes in their codebase, because it is helpful to not repeat the sans mistake again.